x-pack/test/functional/fixtures/kbn_archiver/lens/default.json

```diff
@@ -85,7 +85,7 @@
           "meta": {
             "alias": null,
             "disabled": false,
-            "indexRefName": "filter-index-pattern-0",
+            "index": "filter-index-pattern-0",
```

Comment:
Not yet tested.

Reply:
Do not edit the saved objects in the fixtures - those will go through the migration process, so it's an additional check whether everything works.

Reply:
Okay, thanks @flash1293 ! Will revert.